controllers/api/v1/keyword.go

```diff
@@ -0,0 +1,51 @@
+package apiv1
+
+import (
+	"net/http"
+
+	"google-scraper/models"
+	"google-scraper/serializers"
+
+	"github.com/beego/beego/v2/adapter/context"
+	"github.com/beego/beego/v2/adapter/utils/pagination"
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+)
+
+var sizePerPage int
+
+type Keyword struct {
+	baseAPIController
+}
+
+func init() {
+	var err error
+	sizePerPage, err = web.AppConfig.Int("sizePerPage")
+	if err != nil {
+		logs.Error("Retrieving sizePerPage failed: ", err)
```

Comment:
What is gonna happen if `sizePerPage` has no value? Is it gonna fail this controller? If yes, should we handle something here instead of just log? I'm not sure if beego `logs.error` will exit the app.

This is the same as @Lahphim comment.

Reply:
> What is gonna happen if sizePerPage has no value? Is it gonna fail this controller?

@gutakk Hi gut, I checked it. It does not fails if sizePerPage has no value. I that case it render/response all the keywords at once